src/features/bulk/bulk/__mocks__/bulk.ts

```diff
@@ -1,8 +1,9 @@
-export const onCloseMock = jest.fn();
-export const onMetadataSubmitMock = jest.fn().mockResolvedValue('some-value')
-export const onFileSuccessMock = jest.fn().mockResolvedValue('some-value')
-export const onFileRemoveMock = jest.fn().mockResolvedValue('some-value')
-export const onCancelMock = jest.fn().mockResolvedValue('some-value')
-export const getPostUrlMock = jest.fn();
 export const getHeadersMock = jest.fn();
+export const getPostUrlMock = jest.fn();
```

Comment:
Mocks like these don't deserve being collected here IMO. I think such oneliners should just be declared where they are used

Reply:
I changed this in your previous PR feedback: https://github.com/Amsterdam/bmi-dms-upload/pull/60/files/54d57aa15c435edc46f21f5e87669dfe48feea5f#diff-c1463af751f61094662a1f12a60c562a1af8be15996ad35737a65c5f276f2baa